lemur/notifications/cli.py

```diff
@@ -111,13 +112,15 @@ def security_expiration_summary(exclude):
 
 def notify_expiring_deployed_certificates(exclude):
     """
-    Attempt to find any certificates that are expiring soon but are still deployed,
-    and notify the certificate owner.
+    Attempt to find any certificates that are expiring soon but are still deployed, and notify the certificate owner.
+    This information is retrieved from the database, and is based on the previous run of
+    identity_expiring_deployed_certificates.
     """
     status = FAILURE_METRIC_STATUS
     try:
         print("Starting to notify owners about certificates that are expiring but still deployed!")
-        success = send_expiring_deployed_certificate_notifications(exclude)
+        certificates = get_expiring_deployed_certificates(exclude).items()
```

Comment:
I had to restructure this due to a circular dependency issue in Lemur caused by #3536:

1. In #3536, I added a dependency from `messaging.py` on `certificates/service.py`  in order to find all relevant expiring deployed certs
2. In this PR (#3583), I added a (circular) dependency from `certificates/service.py` on `messaging.py` in order to send a notification when revocation occurs

This seemed like the simplest way to solve the issue with minimal code changes.